src/races.py

```diff
@@ -288,7 +327,7 @@ async def undone(self, ctx):
         except KeyError:
             await ctx.channel.send("Key Error in 'undone' command")
 
-    @commands.command()
+    @commands.command(aliases=['ff'])

```

Comment:
I just saw what happened to the other PR. I'll revert this portion so that the async commands are not broken.